src/ElmSharp/ElmSharp/PreloadedWindow.cs

```diff
@@ -46,6 +50,25 @@ public void InitializeBaseLayout()
             conformant.SetContent(BaseLayout);
         }
 
+        public void CreateWidgets()
```

Comment:
I don't think this method creates widgets. So, how do you feel about renaming? e.g. WarmupXXX , ...

Reply:
Agree. I think we can go with No.1. what do you think? 
1. WarmupWidgets()
2. PreloadWidgets()
3. PrepareWidgets()

Reply:
Those are (whatever) better than previous one for me.